docs/.vuepress/config.js

```diff
@@ -1,78 +1,101 @@
+const glob = require("glob");
+const markdownIt = require("markdown-it");
+const meta = require("markdown-it-meta");
+const fs = require("fs");
+const _ = require("lodash");
+
+const sidebar = (directory, array) => {
+  return array.map(i => {
+    const children = _.sortBy(
+      glob
+        .sync(`./${directory}/${i[1]}/*.md`)
+        .map(path => {
+          const md = new markdownIt();
+          const file = fs.readFileSync(path, "utf8");
+          md.use(meta);
+          md.render(file);
+          const order = md.meta.order;
+          return { path, order };
+        })
+        .filter(f => f.order !== false),
+      ["order", "path"]
+    )
+      .map(f => f.path)
+      .filter(f => !f.match("readme"));
+    return {
+      title: i[0],
+      children
+    };
+  });
+};
+
 module.exports = {
-  title: "Cosmos SDK Documentation",
-  description: "Documentation for the Cosmos SDK and Gaia.",
-  ga: "UA-51029217-2",
-  dest: "./dist/docs",
-  base: "/docs/",
-  markdown: {
-    lineNumbers: true
+  title: "Cosmos SDK",
+  base: process.env.VUEPRESS_BASE || "/",
+  locales: {
+    "/": {
+      lang: "en-US"
+    },
+    "/ru/": {
+      lang: "ru"
+    },
+    "/kr/": {
+      lang: "kr"
+    },
+    "/cn/": {
+      lang: "cn"
+    }
   },
   themeConfig: {
     repo: "cosmos/cosmos-sdk",
-    editLinks: true,
     docsDir: "docs",
-    docsBranch: "master",
-    editLinkText: "Edit this page on Github",
-    lastUpdated: true,
-    algolia: {
-      apiKey: "a6e2f64347bb826b732e118c1366819a",
-      indexName: "cosmos_network",
-      debug: false
-    },
-    nav: [
-      { text: "Back to Cosmos", link: "https://cosmos.network" },
-      { text: "RPC", link: "https://cosmos.network/rpc/" }
-    ],
-    sidebar: [
-      {
-        title: "Overview",
-        collapsable: true,
-        children: [
-          "/intro/",
-          "/intro/why-app-specific",
-          "/intro/sdk-app-architecture",
-          "/intro/sdk-design"
-        ]
-      },
-      {
-        title: "Tutorial",
```

Comment:
need confirmation is this is correct? also if it is then how do we get the tutorial docs up?

Reply:
Not sure. I didn't see tutorial in the sidebar that @gamarin2 and I discussed. The idea is probably to have it somewhere else along with `hellochain` tutorial. Perhaps, http://tutorial.cosmos.network

We can add tutorial files back and include it in the sidebar in the meantime.